packages/terra-abstract-modal/CHANGELOG.md

```diff
@@ -1,7 +1,9 @@
 # Changelog
 
 ## Unreleased
-
+  * Changed
```

Comment:
nit:
```suggestion

  * Changed
```

Reply:
I did not accept the suggestion as it had extra line but made changes manually for both CHANGELOG files.